eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -83,7 +83,15 @@ jobs:
       continueOnError: ${{ parameters.continueOnError }}
     - script: $(Python) $(PerformanceDirectory)/scripts/ci_setup.py $(SetupArguments)
       displayName: Run ci setup script
-    # Run perf testing in helix
+    - script: xcopy $(PerformanceDirectory)\scripts $(WorkItemDirectory)\ScenarioCorrelation\scripts\/e && xcopy $(PerformanceDirectory)\src\scenarios\shared $(WorkItemDirectory)\ScenarioCorrelation\shared\/e && xcopy $(PerformanceDirectory)\src\scenarios\staticdeps $(WorkItemDirectory)\ScenarioCorrelation\staticdeps\/e
```

Comment:
Does the `/e` argument need a preceding space?

Reply:
Nope. Works fine. :)